odd-platform-specification/components.yaml

```diff
@@ -820,8 +817,6 @@ components:
           type: string
         namespace_name:
           type: string
-        token:
```

Comment:
If you removed token from here, may be remove it from DataSourceUpdateFormData as well?